db/migrate/20200119214529_add_organization_info_updated_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizationInfoUpdatedAtToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :organization_info_updated_at, :datetime, default: "2017-01-01 05:00:00"
```

Comment:
Just a note, I think this datetime doesn't need a default :)

Reply:
@rhymes I took example from `github_repos_updated_at` on users which also has this default date :)

Reply:
I understand but adding columns with a default to big tables in PostgreSQL is a very slow operation because it requires rewriting the whole table.

My suggestion is to either remove the default, so that columns will be null (which they already are BTW so we could have the odd situation where some rows are null and some have the default) or add `null: false` so that PostgreSQL 11 can run the operation in constant time.

My preference would be to remove the default, since semantically it's not important: being this a new feature the concept of `NULL` means "the user has never updated the organization info", if there's a value we know they've updated it at least once.

Hope this makes it clearer :)

Reply:
@rhymes Oh ok, now I understand. Thank you very much for this detailed explanation, very interesting!

Sorry when you said just a note I thought you meant it's unnecessary but it doesn't do any harm to leave it like that. 

I will fix that soon :)

Reply:
Thank you @seb1441! Sorry I didn't explain in the beginning :-)

Reply:
@rhymes Fixed!
Btw, the reason I just did multiple force pushes is because I had pushed the new commit with my work git user by mistake and I changed it back to my personal user.

Reply:
No problem, thanks for the heads up. I just opened the build and saw there are a few tests failing: https://travis-ci.com/thepracticaldev/dev.to/builds/145237630